### PR TITLE
CA-161 timezones

### PR DESCRIPTION
**Pull request by katiemcgoff**
- store user timezone upon registration
- display local time for publication date

### CODE REVIEW

File: screens/LoginScreen.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ export default function LoginScreen({navigation}) {
         authContext.setUser(data.data);
         authStorage.storeUser(data.data);
       })
-      .catch(() => {
+      .catch((err) => {
+        console.log(err.response);
```
Comment (katiemcgoff) [inline screens/LoginScreen.js:42]: unrelated change for easier debugging

**Review by JaycobDuffel — APPROVED** [review]

**Review by DawsonDease — APPROVED** [review]
